src/main/java/com/esri/samples/scene/get_elevation_at_a_point/GetElevationAtAPointSample.java

```diff
@@ -0,0 +1,179 @@
+            Surface surface = new Surface();
+            surface.getElevationSources().add(new ArcGISTiledElevationSource("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+            scene.setBaseSurface(surface);
+
+            // create a point symbol to mark where elevation is being measured
+            SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 3.0f);
+
+            // create a text symbol to display the elevation of selected points
+            TextSymbol elevationTextSymbol = new TextSymbol(20,null,0xFFFF0000,TextSymbol.HorizontalAlignment.CENTER,TextSymbol.VerticalAlignment.MIDDLE);
+
+            // create a graphics overlay
+            graphicsOverlay = new GraphicsOverlay(GraphicsOverlay.RenderingMode.DYNAMIC);
+            graphicsOverlay.getSceneProperties().setSurfacePlacement(LayerSceneProperties.SurfacePlacement.RELATIVE);
+            sceneView.getGraphicsOverlays().add(graphicsOverlay);
+
+            // add a camera and initial camera position
+            Camera camera = new Camera(28.42, 83.9, 10000.0, 10.0, 80.0, 0.0);
+            sceneView.setViewpointCamera(camera);
+
+            // create listener to handle clicked
+            sceneView.setOnMouseClicked(event -> {
+
+                // get the clicked screenPoint
+                Point2D screenPoint = new Point2D(event.getX(), event.getY());
+                // convert the screen point to a point on the surface
+                Point relativeSurfacePoint = sceneView.screenToBaseSurface(screenPoint);
+
+                // check that the point is on the surface, and primary button was clicked
+                if (relativeSurfacePoint != null && event.getButton() == MouseButton.PRIMARY && event.isStillSincePress()) {
+
+                    // clear any existing graphics from the graphics overlay
+                    graphicsOverlay.getGraphics().clear();
+
+                    // construct a polyline to use as a marker
+                    PolylineBuilder polylineBuilder = new PolylineBuilder(relativeSurfacePoint.getSpatialReference());
+                    Point baseOfPolyline = new Point(relativeSurfacePoint.getX(), relativeSurfacePoint.getY(), 0);
+                    polylineBuilder.addPoint(baseOfPolyline);
+                    Point topOfPolyline = new Point(baseOfPolyline.getX(), baseOfPolyline.getY(), 750);
+                    polylineBuilder.addPoint(topOfPolyline);
+                    Polyline markerPolyline = polylineBuilder.toGeometry();
+                    Graphic polylineGraphic = new Graphic(markerPolyline, lineSymbol);
+                    graphicsOverlay.getGraphics().add(polylineGraphic);
+
+                    // create a point to place the elevationTextSymbol above the polyline
+                    Point textSymbolPosition = new Point(topOfPolyline.getX(), topOfPolyline.getY(), topOfPolyline.getZ()+100);
```

Comment:
Remove this line and change the text symbol's vertical alignment to `BOTTOM`. Then reuse `topOfPolyline` for the text symbol geometry.